.github/workflows/scan-build.yml

```diff
@@ -13,24 +13,19 @@ jobs:
     steps:
       - name: Install Dependencies
         run: |
-          dnf -y install $COMPILER automake libtool pkgconf-pkg-config \
-              autoconf-archive git openssl-devel clang-analyzer
+          dnf -y install $COMPILER meson pkgconf-pkg-config \
```

Comment:
sounds like this $COMPILER variable is some spurious remnant, probably froma a copy of an old build.yml that was used to jump-start the scan-build action.
I think we can drop it, and/or, if needed, replace it with clang, but I guess clang-analyzer in the next line already pulls in all the required deps.